libs/domain/content/article/article.component.ts

```diff
@@ -32,15 +30,23 @@ export class ArticleComponent extends ContentMixin<ContentArticleOptions>(
     this.contextController.get<string>(ArticleContext.Id)
   );
 
-  protected $article = computed(() => {
+  protected $data = computed(() => {
     const article = this.$articleContext();
 
     return article
       ? this.contentService.get({ article, entities: this.$options().entities })
       : of(null);
   });
 
-  protected override render(): TemplateResult {
-    return html`<oryx-text .content=${this.$article()?.article}></oryx-text> `;
+  protected override render(): TemplateResult | void {
+    const data = this.$data();
+
+    if (!data?.article) {
+      return;
+    }
+
+    return html`<oryx-text
+      .content=${marked.parse(data.article)}
```

Comment:
What about adding a few tests?

Reply:
I will cover it fully in sotryblok integration. I guess I will change it little bit